db/schema.rb

```diff
@@ -95,6 +112,10 @@
     t.string "email_text"
     t.string "receipt_url"
     t.string "name"
+    t.bigint "purchaser_id"
```

Comment:
This should probably be nonnullable

Reply:
rails default `null` to true

https://api.rubyonrails.org/v6.0.3/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference